### DISCUSSION TITLE
Tab key goes to Signal report

### DISCUSSION THREAD

**Discussion by GregTroar** [magicbug/Cloudlog#3083] [Ideas / Feature Requests]
Hello,

Is there a way to have the Tab key to jump from callsign field directly to report sent then report received ?

Thanks

**Comment by phl0** [comment]
> Hello,
> 
> Is there a way to have the Tab key to jump from callsign field directly to report sent then report received ?
> 
> Thanks

Something like this: https://github.com/magicbug/Cloudlog/pull/3084 ?

**Reply by GregTroar** [reply]
Oh yeah exactly. Ok I will wait a bit then :)

**Reply by phl0** [reply]
I am thinking about which fields to activate after RSTr. Maybe RSTs, Name, Grid in that order?

**Comment by GregTroar** [comment]
I would do in this order:
- RSTs
- RSTr
- Name
- Location
- Grid
- Comment

**Reply by phl0** [reply]
Sounds good to me. Will adapt accordingly.

**Reply by phl0** [reply]
PR updated. Pls test

**Comment by magicbug** [comment]
While I understand the idea i'm worried about the accessibility would need to make sure that this wouldn't stop screen readers from understanding whats going on

**Comment by phl0** [comment]
I see. Any idea how to check that?

**Comment by DH5DAX** [comment]
Chrome Screen Reader, for example. But as long as there are no tabindex set to -1, they should be perfectly able to handle it. They will read the postive tabindexes first, the non-indexed (or tabindex=0) second.
